frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -24,55 +24,72 @@ describe('Kubernetes resource CRUD operations', () => {
     cy.logout();
   });
 
+  const tableColumnHeaders =
+    'th > .pf-c-table__button > .pf-c-table__button-content > .pf-c-table__text';
+  const detailSectionHeadings = '[data-test-section-heading]';
+  const detailLabels = 'dt';
+  const detailHorizNavTabs = '.co-m-horizontal-nav__menu-item';
```

Comment:
These selectors should be in a view file.

Reply:
fixed